test/Faucet.t.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Test, console} from "forge-std/Test.sol";
+import {Faucet} from "../src/Faucet.sol";
+import {BaseSetup} from "./BaseSetup.t.sol";
+
+contract FaucetTest is Test, BaseSetup {
+    Faucet faucetContract;
+    uint256 claimAmount;
```

Comment:
You use this global variable only once, I guess we can just iniline it then!

Reply:
This variable is being used at lines 53 and 126

Reply:
These lines seem to be empty:
<img width="723" alt="image" src="https://github.com/Karrot-Works/on-chain-whitelist/assets/41180869/b19f2ede-c160-4374-b9bb-e92408ec9ced">


<img width="1063" alt="image" src="https://github.com/Karrot-Works/on-chain-whitelist/assets/41180869/a223c732-c4bf-4496-aac1-8ad82a38365d">